pandas/io/tests/test_sql.py

```diff
@@ -1435,7 +1485,7 @@ def test_transactions(self):
         raise nose.SkipTest("Nested transactions rollbacks don't work with Pandas")
 
 
-class _TestSQLiteAlchemy(object):
+class _TestSQLiteAlchemy(SQLAlchemyMixIn):
```

Comment:
Are these ones needed to inherit from the mixin? As `_TestSQLAlchemy` already does inherit from that